include/net/lwm2m_client_utils.h

```diff
@@ -221,6 +231,11 @@ struct lwm2m_fota_event {
  * return negative error code to cancel the update or positive value to indicate that
  * update should be postponed that amount of seconds.
  *
+ * LWM2M_FOTA_UPDATE_MODEM_RECONNECT_REQ: Request a reconnect and init modem after firmware update.
+ * Callback handler may return 0 to indicate that application is handling the reconnect.
+ * Modem initialization and LwM2M client re-connection needs to be handled outside of the callback
+ * context. Otherwise return -1 to cause device to reboot.
```

Comment:
```suggestion
 * LWM2M_FOTA_UPDATE_MODEM_RECONNECT_REQ: Request a reconnect and initialize the modem after a 
 * firmware update.
 * The callback handler may return 0 to indicate that the application is handling the
 * reconnect.
 * Modem initialization and LwM2M client re-connection needs to be handled outside of the
 * callback context.
 * Otherwise, return -1 to indicate that the device is rebooting.
```